src/frontend/komponenter/Fagsak/Opprett/typer.ts

```diff
@@ -0,0 +1,4 @@
+import { IFelt } from '../../../typer/felt';
+
+export type IFøldselsnummerFelt = IFelt<string>;
```

Comment:
I dont think we need to move these to a separate file. They can just be defined in the interface IStore.

Reply:
Agree, I will do it🙂